cli.js

```diff
@@ -50,15 +51,40 @@ const cli = meow(chalk`
                       containing the app name and arguments for the app
   --open-path         The path with the app a browser should open to
   --port              The port the app should listen on
-  --version           Display the webpack-serve version
+  --require, -r       Preload one or more modules before loading the webpack configuration
+  --version, -v       Display the webpack-serve version
 
 {underline Examples}
   $ webpack-serve ./webpack.config.js --no-reload
   $ webpack-serve --config ./webpack.config.js --port 1337
   $ webpack-serve # config can be omitted for webpack v4+ only
-`);
+`, {
```

Comment:
nix these aliases please. aliases are really only useful in situations where a command is going to be physically typed frequently and flags need to be repeated frequently. the CLI for this module shouldn't be - it'd be kind of bananas for it not to be in an `npm script`.